pkgs/games/osu-lazer/bin.nix

```diff
@@ -1,37 +1,83 @@
-{ appimageTools, lib, fetchurl }:
+{ lib
+, stdenv
+, fetchurl
+, fetchzip
+, appimageTools
+}:
 
-appimageTools.wrapType2 rec {
+let
   pname = "osu-lazer-bin";
-  version = "2023.301.0";
+  version = "2023.305.0";
+  name = "${pname}-${version}";
```

Comment:
Why was this necessary?

Reply:
Nice catch! Nope, `name = "${pname}-${version}";` should be removed in the next update, or you can make a separate PR removing it.

Reply:
It was bothering me because I use `allowUnfreePredicate`, and this causes it to change names (thus giving an unfree error) with each update. Nice to know that it was superfluous.